File: src/main/java/com/monis/ThreadTester2.java
package com.monis;

public class ThreadTester2 {
    public static void main(String[] args) {
        Thread thread = new Thread(){
            @Override
            public void run() {
                for (int i = 0; i < 1000000; i++) {
                    System.out.println(i);
                }
            }
        };
        thread.start();
    }
}

File: src/main/kotlin/com/monis/ThreadTester2.kt
package com.monis

import kotlinx.coroutines.GlobalScope
import kotlinx.coroutines.launch
import kotlin.concurrent.thread
import kotlin.system.measureTimeMillis

fun main(args: Array<String>) {
    var counter = 0
    var ms = measureTimeMillis {
        for (i in 1..1_000_000) {
            GlobalScope.launch {
                counter++
            }
        }
    }
    println(ms)
}